cli/commands/stack/command.go

```diff
@@ -0,0 +1,31 @@
+package stack
+
+import (
+	"github.com/gruntwork-io/terragrunt/cli/commands"
+	"github.com/gruntwork-io/terragrunt/options"
+	"github.com/gruntwork-io/terragrunt/pkg/cli"
+)
+
+const (
+	CommandName = "stack"
+)
+
+func NewFlags(opts *options.TerragruntOptions) cli.Flags {
+	return cli.Flags{
+		commands.NewNoIncludeRootFlag(opts),
```

Comment:
I know this is WIP, but I'm curious about this. Do you think we need this for `stack` commands?